server/tests/utils/test_tokens.py

```diff
@@ -2,8 +2,14 @@
     StopSequenceCriteria,
     StoppingCriteria,
     FinishReason,
+    NextTokenChooser,
 )
 
+from lorax_server.utils.lora import AdapterBatchData
+from lorax_server.pb import generate_pb2
+from lorax_server.models.causal_lm import CausalLM, CausalLMBatch
+from tests.models.test_causal_lm import default_causal_lm, default_causal_lm_batch
```

Comment:
Imports from `tests` can be a little weird. The recommended thing to do here, since these are fixtures, would be to move them into `conftest.py`. Then you can pass them as arguments to your test functions without needing to import them (see usage of `default_pb_parameters` as an example).

Reply:
Yes, looks like the current failing test is failing due to this issue.